liblangutil/Scanner.cpp

```diff
@@ -73,6 +73,7 @@ std::string to_string(ScannerError _errorCode)
 		case ScannerError::IllegalToken: return "Invalid token.";
 		case ScannerError::IllegalHexString: return "Expected even number of hex-nibbles.";
 		case ScannerError::IllegalHexDigit: return "Hexadecimal digit missing or invalid.";
+		case ScannerError::IllegalBinDigit: return "Binary digit missing or invalid.";
```

Comment:
Needs a syntax test.  I suggest putting it [here](https://github.com/ethereum/solidity/pull/15046/files#diff-4d60ccc5cfd8add6618b7297db754ba098d4792a1e62fd9be5628c0ed4d6941b) along with `0b20`.

Reply:
I decided to make a new test file at https://github.com/ethereum/solidity/pull/15046/commits/c1651ac50419213a59f05c5e4454a0ff19aa873e because if parser error occurs, parser stops and does not progress further to search additional errors.